src/decanter/core/enums/numerical_group_by_methods.py

```diff
@@ -0,0 +1,7 @@
+'''available numerical group by methods supported by corex'''
+from enum import Enum
+
+class NumericalGroupByMethod(Enum):
+    sum = 'sum'
+    mean = 'mean'
+    count = 'count'
```

Comment:
count is not a numerical group by method. It is a categorical group by method. Should have a separate enum for that, so 

`NumericalGroupByMethod: sum, mean`

`CategoricalGroupByMethod: count, mode`

Reply:
ok sounds good. I will make that change